front/app/api/idea_count/types.ts

```diff
@@ -7,8 +7,8 @@ export interface IQueryParameters {
   projects?: string[] | null;
   phase?: string | null;
   topics?: string[];
-  ideaStatusId?: string;
-  feedbackNeeded?: boolean;
+  idea_status_id?: string;
+  feedback_needed?: boolean;
   assignee?: string;
```

Comment:
@IvaKop More for my own knowledge - using the underscores isn't really a convention on the FE right? In older resources, we mapped a variable like `feedbackNeeded` to the `feedback_needed` format when we needed to. Is it preferred that we use the exact BE variable name on the FE though? Even for input props?

E.g In `GetIdeas` many variable were mapped like:
`
    project_publication_status: this.props.projectPublicationStatus,`

Reply:
Admittedly, it's a bit confusing. For component props we prefer snake case `feedbackNeeded`. For the request type though, we use the underscore which is how the back-end expects the query parameters to be passed to it.